src/components/Main.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
```

Comment:
``Component`` is never used here, yes? Maybe just
```jsx
import React from 'react';
```

Reply:
Correct. Component isn't used at the moment. This was me purely testing a functional component. Will remove Component for now until we start UI and refactor the existing components.